docs/manual/grelfunctions.md

```diff
@@ -447,6 +447,10 @@ As of OpenRefine 3.4.1, uniques() reorders the array items it returns; in 3.4 be
 
 ## Date functions {#date-functions}
 
+:::info
+Date/times without timezone info were interpreted as **local** up until May 2018 when OpenRefine 3.0 was released, at which point they were switched from **local** to **UTC**.  One benefit of this change was to introduce consistency and reproducibility when working collaboratively and sharing projects across timezones.  Discussed in issue [#6009](https://github.com/OpenRefine/OpenRefine/issues/6009)
+:::
+
 ###### now() {#now}
 
 Returns the current time according to your system clock, in the [ISO 8601 extended format](exploring#data-types) (converted to UTC). For example, 10:53am (and 00 seconds) on November 26th 2020 in EST returns [date 2020-11-26T15:53:00Z].
```

Comment:
Not related to this PR, but since it caught my eye, ISO 8601 isn't involved. Although this does the same thing as before, due to the changes in `toString()`, the effective date/time is different.

Reply:
I don't quite understand.  This is the `now()` function directly, which returns a date-time with an offset from UTC/Greenwich in the ISO-8601 calendar system, according to the Java docs.  So I think this reads OK as is?  But I think your point is that we need to improve the `toString()` documentation more fully?  I can do that in this PR also.

Reply:
Actually, that's wrong... `now()` will always return the date-time from the users system clock in the specified time-zone (which is `Z` [according to our code](https://github.com/OpenRefine/OpenRefine/blob/ebae7adef85322ddd53b749e10fde8eda8bcd8f7/main/src/com/google/refine/expr/functions/date/Now.java#L48) , which means Zulu time, which means we return the users system time without any offset.)

So should `now()` docs stay as is, because it does return the users system clock in ISO 8601 format?

Reply:
It's only semantics, but the date handling is complicated enough without leading people astray. ISO 8601 dates are represented using strings. What the `now()` function returns is an internal OpenRefine datetime object. The square brackets around "[date 2020-11-26T15:53:00Z]" and the leading "date" prefix indicate that this is the default `toString()` rendering for an internal date object, not a string. The expression `now().toString() would give "2020-11-26T15:53:00Z" for the above example. Using OpenRefine 2.8, `now()` returns a date object rendered as "[date 2024-08-20T17:16:42Z]" while `now().toString()` returns "Aug 20, 2024" using my default system locale and now().toString("hh:mm a z") return "05:20 PM EDT" which matches the current local wall clock time.

In addition to date to string conversions, import/export from spreadsheet formats like Excel and OpenOffice behave differently as well.

Reply:
Wait, so you want to document that OpenRefine prior to 3.0 handled this differently?  If that is your concern, I think it's less important how prior 3.0 handled things and just document how they currently work over the last few versions.  Our surveys show that everyone is using at least OpenRefine 3.7+ ?

Reply:
The whole point of #6009 is documenting (instead of fixing) the date handling regression. If that's not what this is addressing, you should remove the tag. 

The discussion of `now()` is separate. It doesn't return an ISO 8601 formatted string. It returns a datetime.

Reply:
@tfmorris Can you express how you would word things?
>I think the description needs fleshing out a little bit with a general discussion of the change and compatibility implications, as well as specific references near the affected functions (mostly toDate() and toString(), I think).